vaccine_feed_ingest/runners/mn/gov/normalize.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import os
+import pathlib
+import sys
+from typing import List, Optional
+
+from pydantic import ValidationError
+from vaccine_feed_ingest_schema import location as schema
+
+from vaccine_feed_ingest.utils.log import getLogger
+
+logger = getLogger(__file__)
+
+
+def _get_availability(site: dict) -> Optional[schema.Availability]:
+    if not site["attributes"]["TestingRequirements"]:
+        return None
+
+    requirements = site["attributes"]["TestingRequirements"].split("|")
+
+    return schema.Availability(
+        appointments=True if "Appointment" in requirements else None,
+        drop_in=True if "No appointment necessary" in requirements else None,
+    )
+
+
+def _get_id(site: dict) -> str:
+    data_id = site["attributes"]["globalid"]
+
+    # Could parse these from directory traversal, but do not for now to avoid
+    # accidental mutation.
+    site_name = "gov"
+    runner = "mn"
+
+    # Could parse these from the input file name, but do not for now to avoid
+    # accidental mutation.
+    arcgis = "18279cf4ab4740ae92708db437250056"
+    layer = 0
+
+    return f"{runner}_{site_name}:{arcgis}_{layer}_{data_id}"
+
+
+def _get_contacts(site: dict) -> Optional[List[schema.Contact]]:
+    contacts = []
+
+    if phone := site["attributes"]["Phone"]:
+        contacts.append(schema.Contact(phone=phone))
+
+    def cleanup_url(url):
+        if not url:
+            return None
+        if "@" in url:
+            # Some of these are email addresses.
+            # Skipping those for now.
+            return None
+        if url == "COVID-19 Vaccine Information - Hennepin Healthcare":
+            return None
+        if url == "https/:www.Lewisdrug.com":
```

Comment:
A more robust fix for this may be `replace("https/:", "https://")` to catch this typo for other urls, but this works :)